problems/arraysIntro.js

```diff
@@ -64,29 +86,59 @@ function arrayToString() {}
  * @param {Array} array2
  * @returns {string|Array}
  */
-function largerArray() {}
+function largerArray(array1, array2) {
+  if (array1.length > array2.length){
+    return array1
+  }else if (array1.length < array2.length){
+    return array2
+  } else{
+    return "They are the same size"
+  }
+  
+}
 
 /**
  * Takes in an array and returns the middle element.
  * If the array is even in length return the middle two elements in their own array.
  * @param {Array} array
  * @returns {*|Array}
  */
-function middleElement() {}
+function middleElement(array) {
```

Comment:
Very cool to use slice. Pro tip. You're using array.length / 2 in three places. For cleaner code pull out a variable assigned to that and reuse that variable.